backend-node/routes/vonage.js

```diff
@@ -80,7 +80,22 @@ vonageRoutes.get('/conversations', async (req, res) => {
   const pool = new Pool({ connectionString: process.env.postgresDatabaseUrl });
   pool.connect(async (err, client, done) => {
     if (err) throw err
-    const vonageConversations = await Data.conversations.getAllForUser(client, req.user.user_id);
+    const vonageConversations = await Data.conversations.getAllForUser(client, req.user.user_id, true);
+    client.release();
+    return res.status(200).json(vonageConversations);
+  });
+  pool.end();
+});
+
+vonageRoutes.get('/calls', async (req, res) => {
+  const jwt = fromHeaderOrQuerystring(req);
+  if (!jwt || !req.user || !req.user.user_id) {
+    return res.status(403).json("Unauthorised");
+  }
+  const pool = new Pool({ connectionString: process.env.postgresDatabaseUrl });
+  pool.connect(async (err, client, done) => {
+    if (err) throw err
+    const vonageConversations = await Data.conversations.getAllForUser(client, req.user.user_id, false);
```

Comment:
Small thing. Could we name it `vonageCalls` or something to kind of differentiate it from the chats so someone reading can know what it is that they are getting back.
```suggestion
    const vonageCalls = await Data.conversations.getAllForUser(client, req.user.user_id, false);
```